src/classes/CCON_Faculty_TDTM.cls

```diff
@@ -53,9 +53,8 @@ public with sharing class CCON_Faculty_TDTM extends TDTM_Runnable {
         Id facultyRecType = (Id)UTIL_Describe.getFacultyConnectionRecType();
 
         List<Course_Offering__c> courseOfferingsToUpdate = new List<Course_Offering__c>();
-        List<Course_Enrollment__c> courseConnsBeforeInsert = new List<Course_Enrollment__c>();
```

Comment:
FYI, I removed this because it wasn't used anywhere.

Reply:
i love deleting code :)